nilearn/datasets/utils.py

```diff
@@ -1,9 +1,15 @@
 """Downloading NeuroImaging datasets: utility functions."""
 import os
-from pathlib import Path
+from warnings import warn
 
 from .._utils import fill_doc
 
+_GENERAL_MESSAGE = (
+    "The import path 'nilearn.utils' will deprecated in version 0.13. "
+    "Importing from 'nilearn.utils' will be possible "
+    "at least until release 0.13.0."
+)
```

Comment:
that should reference 'nilearn.datasets.utils' no?

Reply:
indeed. 😅